client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -143,11 +151,22 @@ class Dashboard extends Component {
 			return <SetupTasksView onFinished={ this.onStoreSetupFinished } site={ selectedSite } />;
 		}
 
+		let manageView = null;
```

Comment:
you could put `let manageView = <ManageOrdersView site={ selectedSite } />;` here and drop the `else` below.